differ/differ.go

```diff
@@ -621,9 +637,18 @@ func processReportsByCluster(config *conf.ConfigStruct, ruleContent types.RulesM
 
 		report, err := storage.ReadReportForClusterAtTime(cluster.OrgID, cluster.ClusterName, cluster.UpdatedAt)
 		if err != nil {
-			ReadReportForClusterErrors.Inc()
-			skippedEntries++
-			log.Err(err).Msg(operationFailedMessage)
+			// is the problem reported already?
+			reportedAlready, readErr := storage.ReadErrorExists(cluster.OrgID, cluster.ClusterName, time.Time(cluster.UpdatedAt))
+			checkReadError(readErr)
+
+			// if not reported, process the error
+			if !reportedAlready {
```

Comment:
to be more idiomatic we could do

```
    if reportedAlready {
        continue
    }
```

to be more go idiomatic

otherwise looks good to me!

Reply:
ah ok ok